.github/workflows/sdk_build.yml

```diff
@@ -0,0 +1,41 @@
+name: Dapr dotnet sdk CI
```

Comment:
Can we change the name to `dapr_dotnet_sdk` to be consistent with cli ci ? also, when you add build badge to README.md, you need to replace space with %20 :) ...